rocAL/rocAL/include/pipeline/master_graph.h

```diff
@@ -50,6 +50,25 @@ THE SOFTWARE.
 #define MAX_NUM_ANCHORS 8732  // Num of bbox achors used in SSD training
 #define MAX_MASK_BUFFER 10000
 
+#if ENABLE_SIMD
```

Comment:
this seems to be missing in tot MIVIsionX. Why is this needed here

Reply:
This portion was a part of [master_graph.cpp](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/8cbd2e86fdff77faf1ce93cb4e7121010142573a/rocAL/rocAL/source/pipeline/master_graph.cpp#L41). 
We have moved the masks as constants into the master_graph.h hence we have moved the include part also from the cpp file into master_graph.h file